Free/functor

```diff
@@ -0,0 +1,15 @@
+    let Functor =
+      https://raw.githubusercontent.com/FormationAI/dhall-bhat/8afbcee6dfd092764df9e9456fe2a2650748dce2/Functor/Type
+
+in    λ(f : Type → Type)
+    →   { map =
+              λ(a : Type)
+            → λ(b : Type)
+            → λ(g : a → b)
+            → λ(m : ./Type f a)
+            → λ(r : Type)
+            → λ(kp : b → r)
+            → λ(kf : f r → r)
+            → m r (λ(x : a) → kp (g x)) kf
+        }
+      : Functor (./Type f)
```

Comment:
And this can be
```dhall
  λ(f : Type → Type)
→ ../../Mu/functor (../../FreeF/Type f) (../../FreeF/bifunctor f)
```

Reply:
Trying this out with the alternative representation; this doesn't quite work because it requires `Functor f`.  We can implement it directly using the `FreeF` implementation, but we just can't do it through `Mu FreeF`, unfortunately :(